app/data/models/application.js

```diff
@@ -16,21 +23,18 @@ module.exports = (sequelize, DataTypes) => {
     },
     data: DataTypes.JSONB,
     claimed: { type: DataTypes.BOOLEAN, defaultValue: false },
-    createdAt: { type: DataTypes.DATE, defaultValue: Date.now() },
-    updatedAt: { type: DataTypes.DATE, defaultValue: null },
-    createdBy: DataTypes.STRING,
-    updatedBy: { type: DataTypes.STRING, defaultValue: null },
+    createdBy: { type: DataTypes.STRING, defaultValue: 'admin' },
+    updatedBy: { type: DataTypes.STRING, defaultValue: 'admin' },
     statusId: DataTypes.SMALLINT
   }, {
     freezeTableName: true,
     tableName: 'application',
     hooks: {
-      afterCreate: async (applicationRecord, _) => {
```

Comment:
The afterCreate has been completely removed. Are we sure that:

`applicationRecord.dataValues.reference = createReference(applicationRecord.id)` and ` await applicationRecord.update(applicationRecord.dataValues)` isn't required?

Reply:
So Im not 100% sure where this is needed but its basically setting a reference ID based on the ID given from database. I believe it should be generated in the set method (setter) I made on the id field of the model definition.
Unit tests show that this reference gets created but Im not sure how to test integrations with other microservices